release_notes.md

```diff
@@ -13,6 +42,7 @@ permalink: /release_notes/
 * Improved naming of auto save games.
 * Set all buttons to select alliances more easily
 * Bug Fix: Fix game crash when game release version is updated
+* Associate TripleA savegames with TripleA
 
 ## 1.9.0.0.3627
```

Comment:
I'm wondering if we should update this to `3635` to be consistent with the build that was _actually_ marked as stable.

Reply:
sounds good